### ISSUE TITLE
git lga command?

### ISSUE THREAD

**Issue by rmglennon** [githubteacher/merge-conflicts#4]
Following along, I wasn't able to use git lga. Here's my terminal response. 

$ git lga
git: 'lga' is not a git command. See 'git --help'.

Did you mean this?
	tag

**Comment by dcam2015** [comment]
I think this was a custom command that the trainer built - it's probably a shortcut for some other code

**Comment by rmglennon** [comment]
@patrickmckenna is lga custom for something like log --graph --all? Hard to follow there when I couldn't execute it myself.

**Comment by patrickmckenna** [comment]
@rmglennon @dcam2015 Sorry for the confusion. I used it to try to  keep mostly merge-related commands on screen. @dcam2015 you're right, it's a Git alias that I defined. You can find background on Git aliases on [this page](http://git-scm.com/docs/git-config).

I defined `git lga` to be `git log --oneline --all --decorate --graph`. As you discovered, if you haven't defined `git lga`  on your machine, you will get the error message mentioned.

Hope that helps!

**Comment by dcam2015** [comment]
No I think it's an alias.  I think there were some similar commands run in the class that created a similar graph  with
```bash
git log --oneline --all --decorate --graph

**Comment by itsmylifesoham** [comment]
just enter > git config --global alias.lga "log --oneline --all --decorate --graph"
